src/nsls2api/main.py

```diff
@@ -56,17 +61,24 @@ def configure_routing():
     print(f"Current working directory: {output}")
 
     # Also include our webpages
-    api.include_router(home.router)
-    api.include_router(diagnostics.router)
-    api.mount("/static", StaticFiles(directory=static_root_absolute), name="static")
-    api.mount(
+    app.include_router(home.router)
+    app.include_router(diagnostics.router)
+    app.mount("/static", StaticFiles(directory=static_root_absolute), name="static")
+    app.mount(
         "/assets",
         StaticFiles(directory=static_root_absolute / "assets"),
         name="assets",
     )
 
+# @api.middleware("http")
```

Comment:
May as well delete this, I suppose.

Good to know that the Starlette version of this is more efficient. Tiled stacks up a _bunch_ of these and probably could due to refactor them.